content/_data/events/2017-11-08-triangle.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Triangle Area JAM"
+date: "2017-11-08T18:00:00"
+link: "https://www.meetup.com/Raleigh-Jenkins-Area-Meetup/events/241684345/"
+---
+We are currently looking for speakers for this Triangle Jenkins Area Meetup. If you or your company would like to present at, host, or sponsor this Meetup please contact Parker Ennis (rennis@cloudbees.com) or Amanda DeLuise (adeluise@cloudbees.com).
```

Comment:
Does not seem to be a good meetup description, it will be truncated AFAIK. Would be better to just write "Program TBA, looking for speakers"